docs/shared/recipes/module-federation/dynamic-mfe-angular.md

```diff
@@ -36,46 +41,97 @@ Here's the source code of the final result for this guide.
 
 ### Create an Nx Workspace
 
-To start with, we need to create a new Nx Workspace. We can do this easily with:
+To start with, we need to create a new Nx Workspace and add the Nx Angular plugin. We can do this easily with:
 
 {% tabs %}
 {% tab label="npm" %}
 
+```{% command="npx create-nx-workspace@latest ng-mf" path="~/" %}
+ NX   Let's create a new workspace [https://nx.dev/getting-started/intro]
+
+✔ Which stack do you want to use? · none
+✔ Package-based monorepo, integrated monorepo, or standalone project? · integrated
+✔ Which CI provider would you like to use? · skip
+✔ Would you like remote caching to make your build faster? · skip
+
+ NX   Creating your v19.7.4 workspace.
+
+✔ Installing dependencies with npm
+✔ Successfully created the workspace: ng-mf.
```

Comment:
I would skip this information since it's not too relevant and the specific version would be outdated soon after. In the rest of the docs, we only display the prompts.
```suggestion
```